ament_cmake_test/cmake/ament_add_test.cmake

```diff
@@ -80,6 +91,26 @@ function(ament_add_test testname)
       list(APPEND cmd_wrapper "${_env}")
     endforeach()
   endif()
+  if(ARG_APPEND_LIBRARY_DIRS)
+    if(WIN32)
+      set(_library_dirs_env_var PATH)
+    endif()
+    if(UNIX AND NOT APPLE)
+      set(_library_dirs_env_var LD_LIBRARY_PATH)
+    endif()
+    if(APPLE)
+      set(_library_dirs_env_var DYLD_LIBRARY_PATH)
+    endif()
```

Comment:
Since all three blocks set the same value should this use `elseif(...)` instead?